scripts/cleanup-braces.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import re
+import sys, os
+
+DEBUG = True
+
+TMP_SUFFIX = ".brace_cleanup"
+
+def debug(s):
+    if DEBUG:
+        print(s)
+
+def fix_brace_o(a, b):
+    astr = a.rstrip()
+    if astr == "":
+        return a, b, False
+    if not astr[-1] == '{':
+        return a, b, False
+    a = astr[:-1].rstrip() + '\n'
+    bstr = b.lstrip()
+    indent = len(b) - len(bstr)
+    indent = max(indent - 2, 0)
+    b = " " * indent + "{ " + bstr
+    return a, b, True
+
+def fix_brace_c(a, b):
+    bstr = b.lstrip()
+    if bstr == "":
+        return a, b, False
+    if not bstr[0] == '}':
+        return a, b, False
+    return "", a.rstrip() + " " + bstr, True
+
+def fix_brace_pair(a, b):
+    anew, bnew, fix = fix_brace_o(a, b)
+    if fix:
+        return anew, bnew, False
+    else:
+        return fix_brace_c(anew, bnew)
+
+def fix_braces(filename):
+    # open files for reading and writing
+    fi = Path(filename).open(mode="r", encoding="utf-8", newline='\n')
+    fo = Path(filename + TMP_SUFFIX).open(mode="w", encoding="utf-8", newline='\n')
+    bo = ""
+    merge = False
```

Comment:
`bo` is overwritten on line 52 anyway.
```suggestion
    merge = False  # indicates when `ao` got merged into `bo`, so we can ignore `ao`.
```